common/middleware/ensure-current-location.js

```diff
@@ -1,9 +1,12 @@
 function ensureCurrentLocation({ locationsMountpath, whitelist = [] } = {}) {
   return (req, res, next) => {
+    const { permissions = [], locations = [] } = req.session.user || {}
```

Comment:
If you move the `ensureCurrentLocation` middleware in `server.js` [server.js#L129-L134](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/544b833225215e50121116c30f2213729cc2afdc/server.js#L129-L134)

```
app.use(
  ensureCurrentLocation({
    locationsMountpath: locationsApp.mountpath,
    whitelist: config.AUTH_WHITELIST_URLS,
  })
)
```

To below the routing middleware [server.js#L139](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/master/server.js#L139)

There is no need to have the empty object fallback:
```
|| {}
```

Reply:
Don't think that would work. That would mean all the app routes are served without this middleware ever being run.